public/components/organisms/footer.html

```diff
@@ -1,85 +1,163 @@
-<div class="component" title="footer" version="1">
-  <style>
-    .footer {
-      padding-top: .75em;
-      background: rgba(232, 236, 241, 1);
-    }
-
-    .footer-main {
-      grid-template-columns: 1fr;
-      grid-template-rows: auto;
-      grid-gap: 25px;
-      padding: 2em 0;
-      background: rgba(54, 56, 57, .92);
-      text-align: left;
-      color: white;
-      font-family: arial;
-    }
-
-    .footer-column {
-      margin: 0 25px;
-    }
-
-    .footer_social {
-      margin-top: 42px;
-    }
-
-    .footer-button {
-      background-color: rgba(34, 157, 187, 1);
-      border: none;
-      color: white;
-      padding: 10px 42px;
-      text-align: center;
-      text-decoration: none;
-      font-size: 16px;
-    }
-
-    .footer_social-icons {
-      padding: 0;
-    }
-
-    .footer_social-icons li {
-      display: inline-block;
-      list-style-type: none;
-      -webkit-user-select: none;
-      -moz-user-select: none;
-    }
-
-    .footer_social-icons li img {
-      width: 30px;
-      height: 30px;
-      margin-right: 10px;
-    }
-
-    @media (min-width: 768px) {
-      .footer-main {
-        display: grid;
-        grid-template-columns: 1fr 1fr 1fr;
-        grid-template-row: auto;
-      }
-
-      .footer-column {
-        margin: 0 50px;
-      }
-    }
-  </style>
-  <markup>
-    <footer class="footer-main">
-      <div class="footer-column">
-        <h4>Example</h4>
-        <button class="footer-button" type="button">Click here!</button>
+<div class="footer">
+  <footer class="footer-container">
+    <div>
+      <hr class="hr-1" />
```

Comment:
The horizontal line at the top is actually part of the site-wide layout, so this isn't needed.